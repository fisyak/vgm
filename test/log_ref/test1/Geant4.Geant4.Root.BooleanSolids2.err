Info in <TGeoManager::TGeoManager>: Geometry VGM Root geometry, VGM Root geometry created
Info in <TGeoManager::SetTopVolume>: Top volume is worldV. Master volume is worldV
