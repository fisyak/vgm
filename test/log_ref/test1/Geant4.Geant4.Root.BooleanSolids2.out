Geant4 geometry will be defined via Geant4 and exported to Root

*************************************************************
 Geant4 version Name: geant4-09-00    (29-June-2007)
                      Copyright : Geant4 Collaboration
                      Reference : NIM A 506 (2003), 250-303
                            WWW : http://cern.ch/geant4
*************************************************************

TestBooleanSolids2 finished
Go to Import
VGM info:   Geant4GM::Factory::Import started ...
VGM info:   Importing isotope: 
VGM info:    Isotope:     H   Z =  1   N =   1   A =   1.01 g/mole
VGM info:   Importing isotope: 
VGM info:    Isotope:     H   Z =  1   N =   2   A =   2.01 g/mole
VGM info:   Importing isotope: 
VGM info:    Isotope:     C   Z =  6   N =  12   A =  12.00 g/mole
VGM info:   Importing isotope: 
VGM info:    Isotope:     C   Z =  6   N =  13   A =  13.00 g/mole
VGM info:   Importing isotope: 
VGM info:    Isotope:  U235   Z = 92   N = 235   A = 235.01 g/mole
VGM info:   Importing isotope: 
VGM info:    Isotope:  U238   Z = 92   N = 238   A = 238.03 g/mole
VGM info:   Importing element: 
VGM info:    Element: Basic ( )   Z = 13.0   N =  27.0   A =  26.98 g/mole
VGM info:   Importing element: 
VGM info:    Element: Nitrogen (N)   Z =  7.0   N =  14.0   A =  14.01 g/mole
VGM info:   Importing element: 
VGM info:    Element: Oxygen (O)   Z =  8.0   N =  16.0   A =  16.00 g/mole
VGM info:   Importing element: 
VGM info:    Element: H (H)   Z =  1.0   N =   1.0   A =   1.01 g/mole
   --->  Isotope:     H   Z =  1   N =   1   A =   1.01 g/mole   abundance:  99.99 %
   --->  Isotope:     H   Z =  1   N =   2   A =   2.01 g/mole   abundance:   0.01 %
VGM info:   Importing element: 
VGM info:    Element: C (C)   Z =  6.0   N =  12.0   A =  12.01 g/mole
   --->  Isotope:     C   Z =  6   N =  12   A =  12.00 g/mole   abundance:  98.93 %
   --->  Isotope:     C   Z =  6   N =  13   A =  13.00 g/mole   abundance:   1.07 %
VGM info:   Importing element: 
VGM info:    Element: enriched Uranium (U)   Z = 92.0   N = 235.3   A = 235.31 g/mole
   --->  Isotope:  U235   Z = 92   N = 235   A = 235.01 g/mole   abundance:  90.00 %
   --->  Isotope:  U238   Z = 92   N = 238   A = 238.03 g/mole   abundance:  10.00 %
VGM info:   Importing element: 
VGM info:    Element: Vacuum ( )   Z =  1.0   N =   1.0   A =   1.01 g/mole
VGM info:   Importing material: 
VGM info:    Material:    Basic     density:  2.700 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   8.893 cm 
   --->  Element: Basic ( )   Z = 13.0   N =  27.0   A =  26.98 g/mole  ElmMassFraction: 100.00 %  ElmAbundance 100.00 %
VGM info:   Importing material: 
VGM info:    Material:      Air     density:  1.290 mg/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength: 285.161 m  
   --->  Element: Nitrogen (N)   Z =  7.0   N =  14.0   A =  14.01 g/mole  ElmMassFraction:  70.00 %  ElmAbundance  72.71 %
   --->  Element: Oxygen (O)   Z =  8.0   N =  16.0   A =  16.00 g/mole  ElmMassFraction:  30.00 %  ElmAbundance  27.29 %
VGM info:   Importing material: 
VGM info:    Material: Scintillator     density:  1.032 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:  42.544 cm 
   --->  Element: C (C)   Z =  6.0   N =  12.0   A =  12.01 g/mole
   --->  Isotope:     C   Z =  6   N =  12   A =  12.00 g/mole   abundance:  98.93 %
   --->  Isotope:     C   Z =  6   N =  13   A =  13.00 g/mole   abundance:   1.07 %  ElmMassFraction:  91.47 %  ElmAbundance  47.37 %
   --->  Element: H (H)   Z =  1.0   N =   1.0   A =   1.01 g/mole
   --->  Isotope:     H   Z =  1   N =   1   A =   1.01 g/mole   abundance:  99.99 %
   --->  Isotope:     H   Z =  1   N =   2   A =   2.01 g/mole   abundance:   0.01 %  ElmMassFraction:   8.53 %  ElmAbundance  52.63 %
VGM info:   Importing material: 
VGM info:    Material:  Uranium     density: 13.610 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   4.358 mm 
   --->  Element: enriched Uranium (U)   Z = 92.0   N = 235.3   A = 235.31 g/mole
   --->  Isotope:  U235   Z = 92   N = 235   A = 235.01 g/mole   abundance:  90.00 %
   --->  Isotope:  U238   Z = 92   N = 238   A = 238.03 g/mole   abundance:  10.00 %  ElmMassFraction: 100.00 %  ElmAbundance 100.00 %
VGM info:   Importing material: 
VGM info:    Material:   Vacuum     density:  0.000 kg/m3   temperature:   2.73 K  pressure:   0.00 atm  RadLength: 204727576.737 pc 
   --->  Element: Vacuum ( )   Z =  1.0   N =   1.0   A =   1.01 g/mole  ElmMassFraction: 100.00 %  ElmAbundance 100.00 %
VGM info:   Importing LV: world
VGM info:   Importing solid: worldS
VGM info:   ImportDaughters for world
VGM info:   Importing LV: solid1
VGM info:   Importing solid: boxS
VGM info:   ImportDaughters for solid1
VGM info:   Importing LV: solid2
VGM info:   Importing solid: consS
VGM info:   ImportDaughters for solid2
VGM info:   Importing LV: intersection_solid1_solid2
VGM info:   Importing solid: intersection_solid1_solid2_S
VGM info:   Imported Boolean solid: 
VGM info:   Boolean  "intersection_solid1_solid2_S"  Intersection: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
VGM info:   ImportDaughters for intersection_solid1_solid2
VGM info:   Importing LV: subtraction_solid1_solid2
VGM info:   Importing solid: subtraction_solid1_solid2_S
VGM info:   Imported Boolean solid: 
VGM info:   Boolean  "subtraction_solid1_solid2_S"  Subtraction: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
VGM info:   ImportDaughters for subtraction_solid1_solid2
VGM info:   Importing LV: union_solid1_solid2
VGM info:   Importing solid: union_solid1_solid2_S
VGM info:   Imported Boolean solid: 
VGM info:   Boolean  "union_solid1_solid2_S"  Union: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
VGM info:   ImportDaughters for union_solid1_solid2
VGM info:   Import positions: 
VGM info:   0th volume: world  
VGM info:      0th daughter  lv=solid1 vgmVol=solid1
VGM info:      1th daughter  lv=solid2 vgmVol=solid2
VGM info:      2th daughter  lv=intersection_solid1_solid2 vgmVol=intersection_solid1_solid2
VGM info:      3th daughter  lv=subtraction_solid1_solid2 vgmVol=subtraction_solid1_solid2
VGM info:      4th daughter  lv=union_solid1_solid2 vgmVol=union_solid1_solid2
VGM info:   1th volume: solid1  
VGM info:   2th volume: solid2  
VGM info:   3th volume: intersection_solid1_solid2  
VGM info:   4th volume: subtraction_solid1_solid2  
VGM info:   5th volume: union_solid1_solid2  
VGM info:   
Geant4_GM_Factory factory solids store: 
   0th solid: Box  "worldS"  hx = 4000mm  hy = 1000mm  hz = 4000mm
   1th solid: Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
   2th solid: Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
   3th solid: Boolean  "intersection_solid1_solid2_S"  Intersection: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
   4th solid: Boolean  "subtraction_solid1_solid2_S"  Subtraction: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
   5th solid: Boolean  "union_solid1_solid2_S"  Union: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
Geant4_GM_Factory factory volumes store: 
   0th: Volume: "world" material "Basic" solid "Box"  5 daughters: "solid1"  "solid2"  "intersection_solid1_solid2"  "subtraction_solid1_solid2"  "union_solid1_solid2"  
   1th: Volume: "solid1" material "Basic" solid "Box"  0 daughters: 
   2th: Volume: "solid2" material "Basic" solid "Cons"  0 daughters: 
   3th: Volume: "intersection_solid1_solid2" material "Basic" solid "Boolean"  0 daughters: 
   4th: Volume: "subtraction_solid1_solid2" material "Basic" solid "Boolean"  0 daughters: 
   5th: Volume: "union_solid1_solid2" material "Basic" solid "Boolean"  0 daughters: 
VGM info:   Geant4GM::Factory::Import finished.
Import finished
Go to export
VGM info:   Exporting isotope: 
VGM info:   Isotope: "H"    Z = 1  N = 1  A = 1.00783 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "H"    Z = 1  N = 2  A = 2.0141 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "C"    Z = 6  N = 12  A = 12 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "C"    Z = 6  N = 13  A = 13.0034 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "U235"    Z = 92  N = 235  A = 235.01 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "U238"    Z = 92  N = 238  A = 238.03 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "Basic"  " "  Z = 13  N = 26.98  A = 26.98 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "Nitrogen"  "N"  Z = 7  N = 14.01  A = 14.01 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "Oxygen"  "O"  Z = 8  N = 16  A = 16 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "H"  "H"  Z = 1  N = 1.00012  A = 1.00795 g/mole
     0th isotope: Isotope: "H"    Z = 1  N = 1  A = 1.00783 g/mole  abundance: 99.9885%
     1th isotope: Isotope: "H"    Z = 1  N = 2  A = 2.0141 g/mole  abundance: 0.0115%
VGM info:   Exporting element: 
VGM info:   Element: "C"  "C"  Z = 6  N = 12.0107  A = 12.0107 g/mole
     0th isotope: Isotope: "C"    Z = 6  N = 12  A = 12 g/mole  abundance: 98.93%
     1th isotope: Isotope: "C"    Z = 6  N = 13  A = 13.0034 g/mole  abundance: 1.07%
VGM info:   Exporting element: 
VGM info:   Element: "enriched Uranium"  "U"  Z = 92  N = 235.3  A = 235.312 g/mole
     0th isotope: Isotope: "U235"    Z = 92  N = 235  A = 235.01 g/mole  abundance: 90%
     1th isotope: Isotope: "U238"    Z = 92  N = 238  A = 238.03 g/mole  abundance: 10%
VGM info:   Exporting element: 
VGM info:   Element: "Vacuum"  " "  Z = 1  N = 1.01  A = 1.01 g/mole
VGM info:   Exporting material: 
VGM info:   Material: "Basic"  density 2.7 g/cm3  radlen 8.89253 cm  intlen 38.8793 cm  state "Solid"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "Basic"  " "  Z = 13  N = 26.98  A = 26.98 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting material: 
VGM info:   Material: "Air"  density 0.00129 g/cm3  radlen 28516.1 cm  intlen 66268 cm  state "Gas"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "Nitrogen"  "N"  Z = 7  N = 14.01  A = 14.01 g/mole  Elm mass fraction: 70 %
   1th element: Element: "Oxygen"  "O"  Z = 8  N = 16  A = 16 g/mole  Elm mass fraction: 30 %
VGM info:   Exporting material: 
VGM info:   Material: "Scintillator"  density 1.032 g/cm3  radlen 42.5442 cm  intlen 70.0657 cm  state "Solid"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "C"  "C"  Z = 6  N = 12.0107  A = 12.0107 g/mole
     0th isotope: Isotope: "C"    Z = 6  N = 12  A = 12 g/mole  abundance: 98.93%
     1th isotope: Isotope: "C"    Z = 6  N = 13  A = 13.0034 g/mole  abundance: 1.07%  Elm mass fraction: 91.4708 %
   1th element: Element: "H"  "H"  Z = 1  N = 1.00012  A = 1.00795 g/mole
     0th isotope: Isotope: "H"    Z = 1  N = 1  A = 1.00783 g/mole  abundance: 99.9885%
     1th isotope: Isotope: "H"    Z = 1  N = 2  A = 2.0141 g/mole  abundance: 0.0115%  Elm mass fraction: 8.52919 %
VGM info:   Exporting material: 
VGM info:   Material: "Uranium"  density 13.61 g/cm3  radlen 0.43583 cm  intlen 15.8772 cm  state "Solid"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "enriched Uranium"  "U"  Z = 92  N = 235.3  A = 235.312 g/mole
     0th isotope: Isotope: "U235"    Z = 92  N = 235  A = 235.01 g/mole  abundance: 90%
     1th isotope: Isotope: "U238"    Z = 92  N = 238  A = 238.03 g/mole  abundance: 10%  Elm mass fraction: 100 %
VGM info:   Exporting material: 
VGM info:   Material: "Vacuum"  density 1e-25 g/cm3  radlen 6.31723e+26 cm  intlen 3.51163e+26 cm  state "Gas"  temperature 2.73 kelvin  pressure 2.96077e-23 atm
   0th element: Element: "Vacuum"  " "  Z = 1  N = 1.01  A = 1.01 g/mole  Elm mass fraction: 100 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Basic"  density 2.7 g/cm3  radlen 8.89253 cm  intlen 38.8793 cm  state "Solid"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "Basic"  " "  Z = 13  N = 26.98  A = 26.98 g/mole  Elm mass fraction: 100 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Air"  density 0.00129 g/cm3  radlen 28516.1 cm  intlen 66268 cm  state "Gas"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "Nitrogen"  "N"  Z = 7  N = 14.01  A = 14.01 g/mole  Elm mass fraction: 70 %
   1th element: Element: "Oxygen"  "O"  Z = 8  N = 16  A = 16 g/mole  Elm mass fraction: 30 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Scintillator"  density 1.032 g/cm3  radlen 42.5442 cm  intlen 70.0657 cm  state "Solid"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "C"  "C"  Z = 6  N = 12.0107  A = 12.0107 g/mole
     0th isotope: Isotope: "C"    Z = 6  N = 12  A = 12 g/mole  abundance: 98.93%
     1th isotope: Isotope: "C"    Z = 6  N = 13  A = 13.0034 g/mole  abundance: 1.07%  Elm mass fraction: 91.4708 %
   1th element: Element: "H"  "H"  Z = 1  N = 1.00012  A = 1.00795 g/mole
     0th isotope: Isotope: "H"    Z = 1  N = 1  A = 1.00783 g/mole  abundance: 99.9885%
     1th isotope: Isotope: "H"    Z = 1  N = 2  A = 2.0141 g/mole  abundance: 0.0115%  Elm mass fraction: 8.52919 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Uranium"  density 13.61 g/cm3  radlen 0.43583 cm  intlen 15.8772 cm  state "Solid"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "enriched Uranium"  "U"  Z = 92  N = 235.3  A = 235.312 g/mole
     0th isotope: Isotope: "U235"    Z = 92  N = 235  A = 235.01 g/mole  abundance: 90%
     1th isotope: Isotope: "U238"    Z = 92  N = 238  A = 238.03 g/mole  abundance: 10%  Elm mass fraction: 100 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Vacuum"  density 1e-25 g/cm3  radlen 6.31723e+26 cm  intlen 3.51163e+26 cm  state "Gas"  temperature 2.73 kelvin  pressure 2.96077e-23 atm
   0th element: Element: "Vacuum"  " "  Z = 1  N = 1.01  A = 1.01 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting volume store: 
VGM info:   Exporting solid: 
VGM info:      Box  "worldS"  hx = 4000mm  hy = 1000mm  hz = 4000mm
VGM info:      0th volume: world   material: Basic
VGM info:   Exporting solid: 
VGM info:      Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
VGM info:      1th volume: solid1   material: Basic
VGM info:   Exporting solid: 
VGM info:      Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
VGM info:      2th volume: solid2   material: Basic
VGM info:   Exporting solid: 
VGM info:      Boolean  "intersection_solid1_solid2_S"  Intersection: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
VGM info:   Exporting solid: 
VGM info:      Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
VGM info:   Exporting solid: 
VGM info:      Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
VGM info:      3th volume: intersection_solid1_solid2   material: Basic
VGM info:   Exporting solid: 
VGM info:      Boolean  "subtraction_solid1_solid2_S"  Subtraction: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
VGM info:   Exporting solid: 
VGM info:      Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
VGM info:   Exporting solid: 
VGM info:      Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
VGM info:      4th volume: subtraction_solid1_solid2   material: Basic
VGM info:   Exporting solid: 
VGM info:      Boolean  "union_solid1_solid2_S"  Union: 
  ConstituentA = Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
  ConstituentB = Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
  Displacement translation (object):  ( 200,  0,  0) mm 
  Displacement rotation (object):  (30,  -45,  0) deg    no reflectionZ
VGM info:   Exporting solid: 
VGM info:      Box  "boxS"  hx = 500mm  hy = 500mm  hz = 500mm
VGM info:   Exporting solid: 
VGM info:      Cons  "consS"  rin1 = 100mm  rout1 = 300mm  rin2 = 200mm  rout2 = 400mm  hz = 1000mm  sphi = 0deg  dphi = 360deg
VGM info:      5th volume: union_solid1_solid2   material: Basic
VGM info:   ExportPlacements for 0th volume world
VGM info:      0th daughter vol = solid1  simple placement: solid1
VGM info:      1th daughter vol = solid2  simple placement: solid2
VGM info:      2th daughter vol = intersection_solid1_solid2  simple placement: intersection_solid1_solid2
VGM info:      3th daughter vol = subtraction_solid1_solid2  simple placement: subtraction_solid1_solid2
VGM info:      4th daughter vol = union_solid1_solid2  simple placement: union_solid1_solid2
VGM info:   ExportPlacements for 1th volume solid1
VGM info:   ExportPlacements for 2th volume solid2
VGM info:   ExportPlacements for 3th volume intersection_solid1_solid2
VGM info:   ExportPlacements for 4th volume subtraction_solid1_solid2
VGM info:   ExportPlacements for 5th volume union_solid1_solid2
Export finished
Go to save Root geometry
